plutus-core/untyped-plutus-core/test/Evaluation/Builtins/SECP256k1.hs

```diff
@@ -0,0 +1,368 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Very thorough tests!  We should probably steal your ideas and re-use them for the existing `verifySignature` function.

Reply:
I aim to be thorough - this isn't the kind of thing you want to be careless with.